pcdsdevices/epics_motor.py

```diff
@@ -89,12 +95,20 @@ class EpicsMotorInterface(FltMvInterface, EpicsMotor):
     EpicsMotor.low_limit_travel.kind = Kind.config
     EpicsMotor.direction_of_travel.kind = Kind.normal
 
+    # Registry of log filters by device name
+    _motion_error_log_filters: dict[str, filter] = {}
+    _moved_in_session: bool
     _egu: str
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._egu = ''
         self.motor_egu.subscribe(self._cache_egu)
+        self._install_motion_error_filter()
+
+    def move(self, position: float, wait: bool = True, **kwargs) -> MoveStatus:
+        self._moved_in_session = True
```

Comment:
Thinking of bluesky, `set` will call `move`, so this is a good spot for the toggle.